flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CoalesceFunctionITCase.java

```diff
@@ -40,23 +41,28 @@
                         .onFieldsWithData(null, null, 1)
                         .andDataTypes(BIGINT().nullable(), INT().nullable(), INT().notNull())
                         .testResult(
-                                coalesce($("f0"), $("f1")),
-                                "COALESCE(f0, f1)",
-                                null,
-                                BIGINT().nullable())
-                        .testResult(
-                                coalesce($("f0"), $("f2")),
-                                "COALESCE(f0, f2)",
-                                1L,
-                                BIGINT().notNull())
-                        .testResult(
-                                coalesce($("f1"), $("f2")), "COALESCE(f1, f2)", 1, INT().notNull())
-                        .testResult(
-                                coalesce($("f0"), 1),
-                                "COALESCE(f0, 1)",
-                                1L,
-                                // In this case, the return type is not null because we have a
-                                // constant in the function invocation
-                                BIGINT().notNull()));
+                                of(
```

Comment:
instead of using a static import in every test, we could simply provide static methods in the upper class, called `resultItem(...)`

Reply:
Sure, I'd prefer though the name `resultSpec()` since `resultItem` is  another more lower level pojo within the class and can cause confusion.